src/qs_scf_post_gpw.F

```diff
@@ -2160,8 +2160,12 @@ SUBROUTINE write_mo_free_results(qs_env)
             rho_total = rho_hard + rho_soft
             CALL get_pw_grid_info(pw_grid=rho_elec_gspace%pw_grid, &
                                   vol=volume)
+            ! rhotot pw coefficients are by default scaled by grid volume
+            ! need to undo this to get proper charge from printed cube
+            CALL pw_scale(rho_elec_gspace, 1/volume)
```

Comment:
This should probably read `1.0_dp / volume` to ensure a double precision division. Same below.